communications.md

```diff
@@ -1,68 +1,89 @@
 # Communications
 
-## Opt-in
+## General guidelines
 
-Opt-in to Slack channels, calendars, or Asana projects, etc., that you are interested in or obligated to. Configure notifications on these tools to suit this mindset:
+Communication is critical in any organization but in doing so, we want to ensure that we are considerate of other people's production processes and time. Therefore, the focus should be on real problem solving and creative endeavors instead of drowning oneself in processes and tools when communicating with one another.
+
+## First line of Communication: Texting
+
+We use a few tools to help team members communicate with one another: Slack, Gmail and Asana. Before you decide to have a face-to-face or voice/video conference, take some time to organize your thoughts, and write them down. Revise it to be as concise as possible. If it's not urgently blocking all of your next tasks, use Slack, Gmail or Asana to contact another team member asynchronously (in an intermittent manner).
+
+Budget communication time. Try to allocate yourself a relatively fixed schedule to initiate, check and respond to messages. To a certain extent, this means try to optimize for the whole block of contiguous time that suits better for your job (knowledge-based, creative endeavors).
+
+#### - Internal Communication Tool: Slack -
```

Comment:
This is external too. See `guest-???` channels we use, as well as `locus-ux` which we shared with the Very Bad Wizards